modules/react/select/stories/Select.stories.mdx

```diff
@@ -34,79 +34,200 @@ yarn add @workday/canvas-kit-react
 
 ### Basic Example
 
-Select should be used in tandem with [Form Field](/components/inputs/form-field/) to meet
-accessibility standards.
+`Select` should be used in tandem with [Form Field](/components/inputs/form-field/) where the
+`inputId` prop matches an `id` on the `Select.Input` to meet accessibility standards. This ensures
+the `label` text from `FormField` is attached to the `Select.Input` and read out as a group for
+voiceover.
 
 <ExampleCodeBlock code={Basic} />
 
+Our example uses [React state](<(https://react.dev/learn/state-a-components-memory)>) to track the
+value of the `Select`.
+
+### Label Position
+
+Set the `labelPosition` prop of the wrapping `FormField` to designate the position of the label
+relative to the `Select`. `labelPosition` accepts the following values:
+
+- `FormField.LabelPosition.Top` (Default)
+- `FormField.LabelPosition.Left`
+
+<ExampleCodeBlock code={LabelPosition} />
+
+### Required
+
+Set the `required` prop of the wrapping `FormField` to `true` to indicate that the field is
+required. Labels for required fields are suffixed by a red asterisk.
+
+<ExampleCodeBlock code={Required} />
+
 ### Disabled
 
-Set the `disabled` prop of the Select to prevent users from interacting with it.
+Set the `disabled` prop of `Select.Input` to prevent users from interacting with it.
 
 <ExampleCodeBlock code={Disabled} />
 
-You may also set the `disabled` prop of the Select Option to prevent users from selecting that
-Option. Note that the value of a Select may be programmatically set to a disabled Option by setting
-the Select's `value` prop to the value of the disabled Option. The disabled Option, however, will
-remain unselectable by the user.
+### Disabled Items
 
-<ExampleCodeBlock code={DisabledOption} />
+To disable items and prevent them from being interactive, you must set two properties.
 
-### Ref Forwarding
+Set `nonInteractiveIds` to an array of `id`s that you want to disable interaction as well as
+searchability when typing into the `Select`.
 
-Select supports [ref forwarding](https://reactjs.org/docs/forwarding-refs.html). It will forward
-`ref` to its underlying `<select>` element.
+You will also need to set the aria attribute `aria-disabled` on the `Select.Item` if the item is
+disabled, otherwise set the attribute to `undefined`. `disabled` in this case is a property on the
+item's object. This ensures that the item is styled as disabled.
```

Comment:
GH isn't allowing me to include a multi-line suggestion for the entire `Disabled Items` section, so I'm pasting it here:

```
### Disabled Items

In order to disable items and prevent users from interacting with them:

1. Set the `nonInteractiveIds` prop of `Select` to an array of disabled item `id`s. This will
   disable interaction for those items and exclude them from type-ahead.

2. Set the `aria-disabled` attribute of all disabled `Select.Item`s to `true`. This ensures the
   items are styled as disabled.

The following example assigns a `disabled` property to the disabled items in the `items` array.
`disabled` is used to populate `nonInteractiveIds` and set `aria-disabled` for the disabled items.
```

I did a few things here:

1. Updated language for consistency.
2. Replaced "searchability" with "type-ahead." I lifted the "type-ahead" nomenclature from the [WAI Listbox](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/) page.
3. Moved the bit about the `disabled` property in `items` to the very end.

Reply:
Nice, i like the clear numbered steps, and the separation of the info of objects.